.hyperlint/style_guide_test.md

```diff
@@ -0,0 +1,11 @@
+
```

Comment:
This file will run the checks (but no recommendations) use it to iterate on the styles / dictionary. It doesn't need to be committed to the repo (but can be).